daemon-tests/tests/happy_path.rs

```diff
@@ -324,6 +324,37 @@ async fn maker_rejects_rollover_of_open_cfd() {
     wait_next_state!(order_id, maker, taker, CfdState::Open);
 }
 
+#[tokio::test]
+async fn maker_rejects_rollover_after_commit_finality() {
+    let _guard = init_tracing();
+    let oracle_data = OliviaData::example_0();
+    let (mut maker, mut taker, order_id) =
+        start_from_open_cfd_state(oracle_data.announcement()).await;
```

Comment:
Should we inline this if we don't need the variable again?

Reply:
I'm actually aiming at storing an open cfd in a DB for the tests, because this is the biggest contributor for our CI / local testing to take so long! it's like 1 min per test at least... In other words, it will soon go away 😁

Reply:
Interesting, looking forward to it!

I think we should also pair this with benchmarking. If there are a few simple optimisations we can do to cut down the setup time, then I'd take that over a persistent test DB I think.

Reply:
the biggest difference seem to be compiler optimisations, e.g. 10s sqlx query in rust debug buids. I agree that we benchmarking should be done & implemented, but I think persistent test DB would trim these numbers even further anyway (+ would allow us to paint a more accurate picture of failure, we already covered this pathway by a separate test, and redoing it for other tests gives no value)